core/src/main/java/hudson/FilePath.java

```diff
@@ -2831,14 +2831,13 @@ private static Integer writeToTar(File baseDir, DirScanner scanner, OutputStream
      * Supports large files > 10 GB since 1.627 when this was migrated to use commons-compress.
      */
     private static void readFromTar(String name, File baseDir, InputStream in) throws IOException {
-        baseDir = baseDir.getCanonicalFile();
 
```

Comment:
```suggestion
```

Reply:
Code as is minimizes diff to before 63cde2d.